babel.config.json

```diff
@@ -19,7 +19,10 @@
           {
             "nonce": "\"k0Mp1lEd\"",
             "importReact": false,
-            "optimizeCss": false
+            "optimizeCss": false,
+            "customModuleOrigins": [
```

Comment:
What setting the config looks like for new modules

Reply:
Yeah all new names need some bikeshedding. Import sources could work — we have an option to explicitly tie the name of the property to the strict api stuff or we can keep it generic.